Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -144,6 +143,11 @@ lemma of_linearEquiv [f : Flat R M] (e : N ≃ₗ[R] M) : Flat R N := by
   have h : e.symm.toLinearMap.comp e.toLinearMap = LinearMap.id := by simp
   exact of_retract _ _ _ e.toLinearMap e.symm.toLinearMap h
 
+/-- If an `R`-module `M` is linearly equivalent to another `R`-module `N`, then `M` is flat
+  if and only if `N` is flat. -/
+protected lemma congr (e : M ≃ₗ[R] N) : Flat R M ↔ Flat R N :=
```

Comment:
Can you check if similar theorems have this name?

Reply:
I have found theorems [Module.Finite.equiv_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Finiteness.html#Module.Finite.equiv_iff) and [Algebra.FiniteType.equiv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/FiniteType.html#Algebra.FiniteType.equiv), so maybe this lemma should be called `Module.Flat.equiv_iff`?

Reply:
I agree.